tutorials/HDF5/Subset-HDF5-hyperspectral.Rmd

```diff
@@ -0,0 +1,241 @@
+---
+title: "Subsetting HDF5 files to make a smaller deliverable"
+code1: HDF5/Subset-HDF5-hyperspectral.R
+contributors: null
+dataProudct: null
+dateCreated: 2020-02-04 14:20:52
+description: Take a large HDF5 file and extract only the information that you need,
+  then save as a new HDF5 file. For an example, we will take an existing hyperspectral
+  dataset (~600Mb) and reduce it in size for subsequent tutorials.
+estimatedTime: 1.0 Hours
+languagesTool: R
+packagesLibraries: rhdf5, raster
+syncID: 884cc5915a3842c1ab6617b060797fab
+authors: Donal O'Leary
+topics: hyperspectral, HDF5, remote-sensing
+tutorialSeries: null
```

Comment:
I'm not sure about this one. I have considered making it the last lesson in the HDF5 series, or the first in the hyperspectral series. I think I prefer making it the last lesson in the HDF5 series, because it seems to be more of an HDF5 skill than a hyperspectral skill to me. I could also see it making sense in the context of the hyperspectral tutorial, because it does affect all of the other tutorials because we are using a slightly different dataset (in particular, with reduced spectral resolution). But I don't have a strong opinion either way, really. I will leave this unchanged for now, pending a consensus opinion.

Reply:
I'm happy to defer to you:  last lesson in the HDF5 series.  Also, we could list is as an "optional" starter tutorial on the landing page for the hyperpectral series but not include it in the listed tutorials.

Reply:
I think this is a great idea! I will add a note to the Trello card for the Hyperspectral tutorial series